omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -89,6 +89,15 @@
             {base_dn, "<%= node['private_chef']['ldap']['base_dn'] || "" %>" },
             {group_dn, "<%= node['private_chef']['ldap']['group_dn'] || "" %>" },
             {login_attribute, "<%= node['private_chef']['ldap']['login_attribute'] || "samaccountname" %>" },
+            %% LDAP Attribute Mappings
+            {display_name_attribute, "<%= node['private_chef']['ldap']['display_name_attribute'] || 'displayname' %>" },
```

Comment:
This is a good question that I also asked myself.  It is because the entire ldap tree is nil when ldap is disabled so we can't set defaults in the attribute file.  Well, we probably good, I was just a bit lazy and did't want to wrap my head around solving that issue in a backwards compatible way.